docs/README.md

```diff
@@ -0,0 +1,624 @@
+Welcome to the *AIM API - Alpha*!
+
+# Alpha Notice
+
+The AIM API is in an "alpha stage" to gather customer feedback. While in alpha, the API may change in backwards incompatible ways to accommodate for fixes and additions. Breaking changes will be communicated to the primary API contact at least 5 business days in advance.
```

Comment:
Al:
> Is this implying Breaking Changes fixes – this previous statement just needs further clarification.
Define “Breaking Changes”.
Also 5 business days in advance of what???

Reply:
I think these stem from confusion about what a [breaking change](https://en.wiktionary.org/wiki/breaking_change) is. A "breaking change" is essentially any change that we make to the API that might cause the code calling our API to no longer work (the client would have to update their code before it will run again).

Unfortunately, there are situations where something is not working properly and to fix it would introduce a change that "breaks" existing client code - the existing code was written with the bad behavior that we need to fix. That makes that example fix a "breaking change", even though it is fixing a preexisting issue.

More likely though, we'll want to add a new feature, but to do so, we'd need to change the structure of the data we receive or return. Making structure changes like that also require client applications to be updated in lockstep - the client code is broken until they update it.

When I say "Breaking changes will be communicated to the primary API contact at least 5 business days in advance", I mean that we will communicate that there will be a breaking change to the API 5 days before we make the change (we deploy the new code to production).

I think this term is fairly well know, but do you all have suggestions on how we could word this better?